tensorflow/python/eager/profiler_test.py

```diff
@@ -47,7 +48,8 @@ def test_profile(self):
     profile_pb.ParseFromString(profile_result)
     devices = frozenset(device.name for device in profile_pb.devices.values())
     self.assertIn('/host:CPU', devices)
-    if config.list_physical_devices('GPU'):
+    if not test.is_built_with_rocm() and config.list_physical_devices('GPU'):
+      # device tracing is not yet supported on the ROCm platform
       self.assertIn('/device:GPU:0', devices)
     events = frozenset(event.name for event in profile_pb.trace_events)
     self.assertIn('three_times_five', events)
```

Comment:
`Traceback (most recent call last):
  File "<embedded stdlib>/unittest/case.py", line 59, in testPartExecutor
    yield
  File "<embedded stdlib>/unittest/case.py", line 605, in run
    testMethod()
  File "/build/work/07c770a45d8136487a6531d96b0354730923/google3/runfiles/google3/third_party/tensorflow/python/eager/profiler_test.py", line 56, in test_profile
    self.assertIn('Mul:Mul', events)
  File "<embedded stdlib>/unittest/case.py", line 1089, in assertIn
    self.fail(self._formatMessage(msg, standardMsg))
  File "/build/work/07c770a45d8136487a6531d96b0354730923/google3/runfiles/google3/third_party/py/absl/testing/absltest.py", line 1767, in fail
    return super(TestCase, self).fail(self._formatMessage(prefix, msg))
  File "<embedded stdlib>/unittest/case.py", line 670, in fail
    raise self.failureException(msg)
AssertionError: 'Mul:Mul' not found in frozenset({'ExecutorState::Process', 'three_times_five', 'mul:Mul', 'SessionRun', 'ExecutorDoneCallback'})`

Reply:
can you please check above error ?

Reply:
@rthadur 

the error you have mentioned
```
AssertionError: 'Mul:Mul' not found in frozenset({'ExecutorState::Process', 'three_times_five', 'mul:Mul', 'SessionRun', 'ExecutorDoneCallback'})
```

seems to be unrelated to the change in this PR.

The change in this PR merely skips the check on line 51/53 on the ROCm platform.

The assert that is firing is on line 56/58, and the reason seems to be a change in the event name ( `mul:Mul` vs `Mul:Mul`). That does not seem related to the change in this PR

Reply:
thank you